BitwardenShared/UI/Auth/ProfileSwitcher/ProfileSwitcherRow.swift

```diff
@@ -213,7 +213,7 @@ struct ProfileSwitcherRow: View {
                         )
                     )
                 }
-                .onLongPressGesture(if: store.state.allowLockAndLogout) {
+                .onLongPressGesture(if: store.state.allowLock || store.state.allowLogout) {
```

Comment:
⚠️ I think we need to be careful here. The purpose of `allowLockAndLogout` is to determine if the view supports showing the action sheet for the lock and logout actions. For example, you can't lock or logout from the autofill screen. So that should take priority over these new properties. But it's maybe also confusing to have `allowLockAndLogout`, `allowLock` and `allowLockout` whereas `allowLockAndLogout` is slightly different than the other two.

I have some similar changes [here](https://github.com/bitwarden/ios/pull/1150/files#diff-741bdaaa7f7c0db8a5fa1222f442067bf1284678be6ea8918e6698c98ffdbb52) to support remove account. Maybe it's easiest to stick with using the `canBeLocked` on `ProfileSwitcherItem`? For the lock accessibility action below, something like:

`if: store.state.allowLockAndLogout && profileSwitcherItem.isUnlocked && profileSwitcherItem.canBeLocked,`